crates/astria-sequencer/src/accounts/action.rs

```diff
@@ -96,6 +96,24 @@ impl ActionHandler for TransferAction {
                 .is_none(),
             "cannot send transfer to bridge account",
         );
+        // ensure that if the sender is a bridge account, that the asset being transferred is not
+        // the same asset as the bridge account's asset. The bridge account should be using
+        // the `BridgeUnlockAction` for that.
```

Comment:
i'm not sure if we need this logic - i think we should either not allow `Transfer`s out of bridge accounts at all, or allow them but not place any restrictions on them. this seems unnecessary, as if the bridge account owner wanted to transfer out the same asset but was restricted from using `Transfer`, they could still just use `BridgeUnlock` and leave the memo empty (or put something random)

I think we should remove this part for now and have a bigger discussion on what actions/accounting should look like